### PR TITLE
Creating a finetuning command that simplify overall finetuning

### PR DESCRIPTION
**Pull request by PonteIneptique**

### CODE REVIEW

File: pie/scripts/group.py [hunk 1]
```diff
@@ -125,5 +125,59 @@ def info(model_file):
     click.echo(repr(m))
 
 
+@pie_cli.command("finetune")
+@click.argument('input_model', type=click.Path(exists=True, file_okay=True, readable=True, dir_okay=False))
+@click.argument('train', type=click.Path(exists=True, file_okay=True, readable=True, dir_okay=False))
+@click.argument('dev', type=click.Path(exists=True, file_okay=True, readable=True, dir_okay=False))
+# @click.option('--test', type=str, default=None)
```
Comment (OrianeN) [inline pie/scripts/group.py:132]: Why did you comment the option for the test set ? 
It is often necessary to declare a new test set when fine-tuning, for instance in case of transfer learning to a new language, or if the original path is no longer accessible.

File: pie/scripts/group.py [hunk 2]
```diff
@@ -125,5 +125,59 @@ def info(model_file):
...
+@click.option('--seed', type=int, default=None)
+@click.option("--batch-size", type=int, default=None)
+@click.option("--epochs", type=int, default=100)
+@click.option("--mode", type=click.Choice(["expand", "skip", "replace"]), default="expand")
```
Comment (OrianeN) [inline pie/scripts/group.py:136]: With the expand mode, I usually used the word/char_min_freq/max_size - could you add them as options as well ?

File: pie/scripts/group.py [hunk 3]
```diff
@@ -125,5 +125,59 @@ def info(model_file):
...
+@click.option("--device", type=str, default="cpu")
```
Comment (OrianeN) [inline pie/scripts/group.py:137]: It could help in avoiding mistakes and avoid typing one more option to choose the device automatically based on what's available (for the default value)

File: pie/scripts/group.py [hunk 4]
```diff
@@ -125,5 +125,59 @@ def info(model_file):
...
+@click.option("--out", type=str, default="model")
```
Comment (OrianeN) [inline pie/scripts/group.py:138]: IIRC, the default model filename when training is a concatenation using the "modelname" setting. Maybe you could do something similar here as well ?

**Review by OrianeN — COMMENTED** [review]
Nice ! Not sure I prefer this option over the config file, but I guess it's a matter of taste.
I'm also discovering the `model._settings` attribute which I didn't find when writing the script [get_pie_model_params.py](https://github.com/OrianeN/PaPie/blob/ON/finetune/scripts/get_pie_model_params.py) ^^

Here are a few suggestions for improvement, what do you think ?
